# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/moz.configure
# Commit: afd62e82fe83
# Full Hash: afd62e82fe8320776e9696e3a8881eaae56b8822
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-11 04:06:39
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Refactor wifi scanning to consolidate generic code r=necko-reviewers,kershaw,valentin
#   
#   Reorganizes the wifi-scanning code and makes more of it platform-generic
#   to ease the transition from polling the wifi to usually scanning only on
#   network changes.  This is essentially just moving files/code around and
# ==============================================================================

diff -r 9c94ab8b34c0 -r afd62e82fe83 toolkit/moz.configure
--- a/toolkit/moz.configure	Wed May 10 19:29:32 2023 +0000
+++ b/toolkit/moz.configure	Wed May 10 19:46:29 2023 +0000
@@ -2992,6 +2992,7 @@
 
 
 set_config("NECKO_WIFI_DBUS", True, when=necko_wifi_dbus)
+set_define("NECKO_WIFI_DBUS", True, when=necko_wifi_dbus)
 
 
 # Frontend JS debug mode
